modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAuthCodeGrantOpenIdTestCase.java

```diff
@@ -317,59 +306,25 @@ public void testClaims() throws Exception {
-
-            JSONObject jwtJsonObject = new JSONObject(new String(Base64.decodeBase64(tokenElements[1])));
-            Assert.assertNotNull(jwtJsonObject.get("exp"), "'exp' value is not included");
-
-            long expValue = Long.valueOf(jwtJsonObject.get("exp").toString());
-            // ratio between these vales is normally 999, used 975 just to be in the safe side
-            Assert.assertTrue(System.currentTimeMillis() / expValue > 975, "'exp time is not in milliseconds'");
-        }
+        JSONObject tokenResponse = introspectToken();
+
+        Assert.assertNotNull(tokenResponse.get("exp"), "'exp' value is not included");
+        long expValue = Long.parseLong(tokenResponse.get("exp").toString());
+        // ratio between these vales is normally 999, used 975 just to be in the safe side
+        Assert.assertTrue(System.currentTimeMillis() / expValue > 975, "'exp time is not in milliseconds'");
     }
 
     @Test(groups = "wso2.is", description = "Validate Authorization Context of jwt Token", dependsOnMethods =
-            "testGetAccessToken")
-    public void testAuthorizationContextValidateJwtToken() throws Exception {
-        String claimURI[] = {OAuth2Constant.WSO2_CLAIM_DIALECT_ROLE};
-        OAuth2TokenValidationRequestDTO requestDTO = new OAuth2TokenValidationRequestDTO();
-        OAuth2TokenValidationRequestDTO_OAuth2AccessToken accessTokenDTO = new
-                OAuth2TokenValidationRequestDTO_OAuth2AccessToken();
-        accessTokenDTO.setIdentifier(accessToken);
-        accessTokenDTO.setTokenType("bearer");
-        requestDTO.setAccessToken(accessTokenDTO);
-        requestDTO.setRequiredClaimURIs(claimURI);
-
-        OAuth2TokenValidationResponseDTO responseDTO = oAuth2TokenValidationClient.validateToken(requestDTO);
-        if (responseDTO != null && responseDTO.getAuthorizationContextToken() != null) {
-            String tokenString = responseDTO.getAuthorizationContextToken().getTokenString();
-
-            String[] tokenElements = tokenString.split("\\.");
-            JSONObject jwtJsonObject = new JSONObject(new String(Base64.decodeBase64(tokenElements[1])));
-            String jwtClaimMappingRoleValues = jwtJsonObject.get(OAuth2Constant.WSO2_CLAIM_DIALECT_ROLE).toString();
-            Assert.assertTrue(jwtClaimMappingRoleValues.contains(","), "Broken JWT Token from Authorization context");
-
-            String[] jwtClaimMappingRoleElements = jwtClaimMappingRoleValues.replaceAll("[\\[\\]\"]", "").split(",");
-            List<String> jwtClaimMappingRoleElementsList = Arrays.asList(jwtClaimMappingRoleElements);
-            Assert.assertTrue(jwtClaimMappingRoleElementsList.contains("Internal/everyone"), "Invalid JWT Token Role " +
-                    "Values");
-        }
+            "testValidateAccessToken")
+    public void testValidateTokenScope() throws Exception {
+        JSONObject tokenResponse = introspectToken();
```

Comment:
Shall we add the missing new line?

Reply:
Fixed with https://github.com/wso2/product-is/pull/20096